services/QuillLMS/config/vite.json

```diff
@@ -2,7 +2,8 @@
   "all": {
     "sourceCodeDir": "client/app",
     "entryPointsDir": "client/app/entrypoints",
-    "watchAdditionalPaths": ["app/assets/stylesheets"]
+    "watchAdditionalPaths": ["app/assets/stylesheets"],
+    "skipCompatibilityCheck": true
```

Comment:
I added this because vite kept stopping compile with the Error "ruby 3.2 may not be compatible with vite_ruby 3.5.0". The gem is in fact compatible. But the warning is causing the code not to compile, so I'm removing the warning for now since during future updates of this gem we will always manually test to make sure everything is compatible.

Reply:
The[ vite ruby docs ](https://vite-ruby.netlify.app/config/index.html#skipcompatibilitycheck)specify that doing this `could lead to hard-to-debug errors such as missing entrypoints in the build.`

Thus, it's worth the extra research into resolving this issue, rather than using the escape hatch.